opencood/hypes_yaml/point_pillar_where2comm.yaml

```diff
@@ -1,73 +1,72 @@
 name: point_pillar_where2comm
 root_dir: '/home/cav/data/v2xset/train'
-validate_dir: '/home/cav/data/v2xset//test'
+validate_dir: '/home/cav/data/v2xset/test'
 
 
 wild_setting:
-  async: false
+  seed: 42
+  async: False
+  async_mode: 'sim'
   async_overhead: 100
-  seed: 20
-  loc_err: false
+  loc_err: False
   xyz_std: 0.2
   ryp_std: 0.2
-  data_size: 1.06 # Mb!!
-  transmission_speed: 27 # Mbps!!
-  backbone_delay: 10 # ms
+  data_size: 1.06  # Mb
+  transmission_speed: 27  # Mbps
+  backbone_delay: 10  # ms
 
-yaml_parser: "load_point_pillar_params"
+yaml_parser: 'load_point_pillar_params'
 train_params:
-  batch_size: &batch_size 4
-  epoches: &epoches 90
+  batch_size: &batch_size 5
```

Comment:
would batch size be an even number better for GPU optimization?

Reply:
I think both even and odd are OK. You can set one that fits the GPU memory. According to my reproduction, a relatively large batch size seems to be better in performance.

Reply:
ok, sounds good to me